pkgs/servers/http/apache-modules/mod_ca/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, pkgconfig, apacheHttpd, openssl, openldap }:
+
+stdenv.mkDerivation rec {
+  pname = "mod_ca";
+  version = "0.2.1";
+
+  src = fetchurl {
+    url = "https://redwax.eu/dist/rs/${pname}-${version}.tar.gz";
+    sha256 = "1pxapjrzdsk2s25vhgvf56fkakdqcbn9hjncwmqh0asl1pa25iic";
+  };
+
+  buildInputs =[pkgconfig apacheHttpd openssl openldap];
+
+  configureFlags =[
+       "--with-apxs=${apacheHttpd.dev}/bin/apxs"
+    ];
+
+  installFlags =[
+           "INCLUDEDIR=$(out)/include"
```

```suggestion
           "INCLUDEDIR=${placeholder ''out''}/include"
```

is preferred these days, since the value will be directly substituted by Nix. `$(out)` relies on `make` substituting it and it might be confusing with regards to escaping.

---

done.